packages/ckeditor5-markdown-gfm/src/html2markdown/html2markdown.ts

```diff
@@ -11,7 +11,7 @@
 
 // Importing types for this package is problematic, so it's omitted.
 // @ts-ignore
-import TurndownService from 'turndown';
+import TurndownService from 'turndown/lib/turndown.browser.es';
```

Comment:
The default `turndown` bundle contains `require('jsdom')`, so I updated the import to use the browser version instead.